configsuite/schema.py

```diff
@@ -82,11 +112,19 @@ def assert_valid_schema(schema):
         raise TypeError("Unknown base container: {}".format(schema))
 
 
-def _assert_valid_schema_level(schema):
+def assert_valid_schema_level(schema, allow_default):
     level_schema = copy.deepcopy(schema)
     if MK.Content in level_schema:
         level_schema.pop(MK.Content)
 
+    if MK.Default in schema:
+        if not allow_default:
+            fmt = "Default value is only allowed for contents in NamedDict"
+            raise ValueError(fmt)
+        if not isinstance(level_schema[MK.Type], types.BasicType):
```

Comment:
This check is also checked by the level schema. I suggest you remove this check ;)

Reply:
Yeppsi! And that made the error message better as well